### PR TITLE
CLI modification

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- Test system modified

#### Any other comments?
@sadrasabouri 
We will include tests for the CLI functions in **version 1.5**

### CODE REVIEW

File: test/error_test.py [hunk 1]
```diff
@@ -61,7 +61,7 @@
 >>> g.plot(color=2, bgcolor=2)
 Traceback (most recent call last):
         ...
-samila.errors.samilaPlotError: Given color/bgcolor type is not supported.
+samila.errors.samilaPlotError: Plotting process can't be Done because data1 is empty. Use generate method first.
```
Comment (sadrasabouri) [inline test/error_test.py:64]: `Done` --> `done`
Alos how are these changed without any changes in `params.py`? Either now or then tests should be failed, right?

**Review by sadrasabouri — COMMENTED** [review]
I noticed a point in the PR, can you please check it?

File: test/error_test.py (hunk 1 above)
Comment (sepandhaghighi) [reply test/error_test.py]: Due to the `IGNORE_EXCEPTION_DETAIL` flag in the `pytest.ini` file, all exception messages were being ignored.

Comment (sepandhaghighi) [reply test/error_test.py]: > `Done` --> `done` 

fix in https://github.com/sepandhaghighi/samila/pull/238/commits/c23ef8dbb76bb8f719c59364403173261276c64e

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**